bundles/org.eclipse.ui.workbench/eclipseui/org/eclipse/ui/internal/WorkbenchMessages.java

```diff
@@ -1,1042 +1,1040 @@
-	public static String FilteredItemsSelectionDialog_separatorLabel;
-	public static String FilteredItemsSelectionDialog_storeError;
-	public static String FilteredItemsSelectionDialog_restoreError;
-	public static String FilteredItemsSelectionDialog_nItemsSelected;
-
-	// AbstractSearcher
-	public static String FilteredItemsSelectionDialog_jobLabel;
-	public static String FilteredItemsSelectionDialog_jobError;
-
-	// GranualProgressMonitor
-	public static String FilteredItemsSelectionDialog_taskProgressMessage;
-	public static String FilteredItemsSelectionDialog_subtaskProgressMessage;
-
-	static {
-		// load message values from bundle file
-		NLS.initializeMessages(BUNDLE_NAME, WorkbenchMessages.class);
-	}
-
-	// Content assist support
-	public static String ContentAssist_Cue_Description_Key;
-
-	// Settings transfer
-	public static String WorkbenchLayoutSettings_Name;
-	public static String WorkbenchSettings_CouldNotCreateDirectories;
-	public static String WorkbenchSettings_CouldNotFindLocation;
-	public static String WorkingSets_Name;
-	public static String WorkingSets_CannotSave;
-	public static String WorkbenchPreferences_Name;
-
-	// StatusDialog
-	public static String WorkbenchStatusDialog_SupportTooltip;
-	public static String WorkbenchStatusDialog_SupportHyperlink;
-	public static String WorkbenchStatusDialog_StatusWithChildren;
-	public static String WorkbenchStatusDialog_SeeDetails;
-	public static String WorkbenchStatusDialog_MultipleProblemsHaveOccured;
-	public static String WorkbenchStatusDialog_ProblemOccurred;
-	public static String WorkbenchStatusDialog_ProblemOccurredInJob;
-
-	public static String StackTraceSupportArea_NoStackTrace;
-	public static String StackTraceSupportArea_CausedBy;
-	public static String StackTraceSupportArea_Title;
-
-	public static String ErrorLogUtil_ShowErrorLogTooltip;
-	public static String ErrorLogUtil_ShowErrorLogHyperlink;
-
-	// WorkingSetConfigurationBlock
-	public static String WorkingSetConfigurationBlock_SelectWorkingSet_button;
-	public static String WorkingSetConfigurationBlock_NewWorkingSet_button;
-	public static String WorkingSetConfigurationBlock_WorkingSetText_name;
-
-	public static String WorkingSetPropertyPage_ReadOnlyWorkingSet_description;
-	public static String WorkingSetPropertyPage_ReadOnlyWorkingSet_title;
-
-	public static String WorkingSetGroup_WorkingSets_group;
-	public static String WorkingSetGroup_WorkingSetSelection_message;
-	public static String WorkingSetGroup_EnableWorkingSet_button;
-
-	public static String FilteredTableBaseHandler_Close;
-
-	// Util
-	public static String Util_List;
-
-	// Zoom change messages
-	public static String Workbench_zoomChangedTitle;
-	public static String Workbench_zoomChangedMessage;
-	public static String Workbench_RestartButton;
-	public static String Workbench_DontRestartButton;
-
-}
+/*******************************************************************************

```

Comment:
Could it be that this PR changes some line-endings unintentionally?

Reply:
Yes, thanks. I had a temporary misconfiguration in my Git CLI. I will update the PR.